### PR TITLE
Migrateshootercode

### PR DESCRIPTION
**Pull request by FrootLoops78**
added different speeds and fixed style issues

### CODE REVIEW

File: src/main/java/frc/robot/subsystems/Flywheel.java [hunk 1]
```diff
@@ -17,9 +17,19 @@
  */
 public class Flywheel extends SubsystemBase {
   private static final TalonSRX m_flywheelMotor = new TalonSRX(Constants.Shooter.FLYWHEEL_MOTOR_ID);
+  private double m_flywheelSpeed = 0;
+
+  /**
+   * constructor for Flywheel.
```
Comment (nicole-pham) [inline src/main/java/frc/robot/subsystems/Flywheel.java]: Capitilization :)

**Review by nicole-pham — DISMISSED** [review]
Fix vertical spacing n fix the capitalization in the spot I commented.

File: src/main/java/frc/robot/RobotContainer.java [hunk 2]
```diff
@@ -31,27 +31,35 @@
...
+  public static final Button normalButton = new JoystickButton(leftJoystick, 2);
+  public static final Button turboButton = new JoystickButton(leftJoystick, 3);
   public static final Button conveyorButton = new JoystickButton(rightJoystick, 2);
   public static final Button intakeButton = new JoystickButton(rightJoystick, 1);
 
   /**
    * The container for the robot. Contains subsystems, OI devices, and commands.
    */
   public RobotContainer() {
+
```
Comment (caffineehacker) [inline src/main/java/frc/robot/RobotContainer.java]: Unnecessary spacing

File: src/main/java/frc/robot/RobotContainer.java [hunk 3]
```diff
@@ -31,27 +31,35 @@
...
+    
     // Configure the button bindings
     configureButtonBindings();
     // m_chassis.setDefaultCommand(m_driveCommand);
     m_chassis.setDefaultCommand(m_encoderReadCommand);
+
```
Comment (caffineehacker) [inline src/main/java/frc/robot/RobotContainer.java]: Same here

File: src/main/java/frc/robot/subsystems/Flywheel.java [hunk 4]
```diff
@@ -17,9 +17,19 @@
...
+   */
+  public Flywheel(double flywheelSpeed) {
+
```
Comment (caffineehacker) [inline src/main/java/frc/robot/subsystems/Flywheel.java]: No need for this blank line

File: src/main/java/frc/robot/subsystems/Flywheel.java [hunk 5]
```diff
@@ -17,9 +17,19 @@
...
+    m_flywheelSpeed = flywheelSpeed;
+
```
Comment (caffineehacker) [inline src/main/java/frc/robot/subsystems/Flywheel.java]: No need for this blank line

**Review by caffineehacker — APPROVED** [review]

**Pull request merged**